spec/services/change_schedule_spec.rb

```diff
@@ -198,13 +204,16 @@
     end
 
     describe ".call" do
+      let(:schedule_identifier) { "ecf-replacement-april" }
+      let!(:new_schedule) { create(:ecf_schedule, schedule_identifier: "ecf-replacement-april", name: "ECF Standard") }
+
       it_behaves_like "changing the schedule of a participant"
 
       it "updates the schedule on the relevant induction record" do
         service.call
         relevant_induction_record = participant_profile.current_induction_record
 
-        expect(relevant_induction_record.schedule).to eq(schedule)
+        expect(relevant_induction_record.schedule).to eq(new_schedule)
```

Comment:
could we add a test that checks the induction programme and the school cohort are changed as well?